CLI/1. User Guides/2. 登入.md

```diff
@@ -0,0 +1,86 @@
+---
+title: 登入 TWCC CLI | en
+tags: Guide, CLI, EN
+GA: UA-155999456-1
+---
+
+{%hackmd @docsharedstyle/twccheader-en %}
+{%hackmd @docsharedstyle/default %}
+
+# Log in to TWCC CLI
+
+After installation, please prepare **Project ID**, **API key (login certificate)**, and then log in to TWCC CLI to start.
+
+## Preparation
+
+### 1. Project ID
+
+Sign in to [TWCC Portal](https://www.twcc.ai/), and check the **Project ID** of the using project on the homepage
+
+![](https://cos.twcc.ai/SYS-MANUAL/uploads/upload_1a1bd7090cfd898384a84b61ef57b747.png)
+
+
+
+
+### 2. API key
+
+- Click **the user name** in the upper right corner, and then click **API Key**.
+
+![](https://cos.twcc.ai/SYS-MANUAL/uploads/upload_00e143d58b5cfee093c4c95d32dd2042.png)
+
+
+
+- Click <i class="fa fa-clone" aria-hidden="true"></i> copy button to get your API key
+
+![](https://cos.twcc.ai/SYS-MANUAL/uploads/upload_ec8357b81ff676799f942907115ab77d.png)
+
+
+
+:::danger
+{%hackmd @docsharedstyle/important-en %}
+ 
+To ensure security, please:
+- **Avoid using the master key to use services, and do not leak the master key out**.
+- Update your key regularly (delete and then create a new one).
+:::
+
+
+## Log in to TWCC CLI
+
+- Enter the following commands, and enter **TWCC API key** and **Project ID** to log in to TWCC CLI
+
+```bash
+$ twccli config init
+```
+
+- Language and other parameter settings
+```bash
+$ export LANG=C.UTF-8
+$ export LC_ALL=C.UTF-8
+$ export PYTHONIOENCODING=UTF-8
+```
+
+## View the user information
+
+View user information, API key, and Poject ID
+
+
+```bash
+$ twccli config whoami
+```
+![](https://cos.twcc.ai/SYS-MANUAL/uploads/upload_357bb8bf5d9208b9e97187cbc56c58c9.png)
+
+
+:::info
+
+{%hackmd @docsharedstyle/note-en %}
+
+**How to switch to another project?**
+
+Use the following command to delete the key, and <ins>[log in to TWCC CLI](#Log-in-to-TWCC-CLI) again</ins>, then you can enter the API key of another project and use it.

```

Comment:
[log in to TWCC CLI](#Log-in-to-TWCC-CLI) > [Sign in with TWCC CLI](#Sign-in-with-TWCC-CLI)

Reply:
then you can enter the API key of another project and use it. >


then you can switch to another project by entering your API key under the project.